test/topology_tasks/test_tablet_tasks.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import asyncio
+import pytest
+
+from test.pylib.internal_types import ServerInfo
+from test.pylib.manager_client import ManagerClient
+from test.pylib.repair import create_table_insert_data_for_repair
+from test.topology.conftest import skip_mode
+from test.topology_experimental_raft.test_tablets import inject_error_on
+from test.topology_tasks.task_manager_client import TaskManagerClient
+
+async def check_and_abort_repair_task(tm: TaskManagerClient, servers: list[ServerInfo], module_name: str):
+    async def get_user_repair_tasks():
+        return [task for task in await tm.list_tasks(servers[0].ip_addr, module_name) if task.kind == "cluster" and task.type == "user_repair" and task.keyspace == "test"]
+
+    async def check_repair_task_status(status):
+        assert status.scope == "table"
+        assert status.kind == "cluster"
+        assert status.type == "user_repair"
+        assert status.keyspace == "test"
+        assert status.table == "test"
+        assert status.is_abortable
+        assert not status.children_ids
+
+    # Wait until user repair task is created.
+    repair_tasks = await get_user_repair_tasks()
+    while not repair_tasks:
+        repair_tasks = await get_user_repair_tasks()
+
+    task = repair_tasks[0]
+    assert task.scope == "table"
+    assert task.keyspace == "test"
+    assert task.table == "test"
+    assert task.state in ["created", "running"]
+
+    status = await tm.get_task_status(servers[0].ip_addr, task.task_id)
+
+    check_repair_task_status(status)
+    assert status.state in ["created", "running"]
+
+    async def wait_for_task():
+        status_wait = await tm.wait_for_task(servers[0].ip_addr, task.task_id)
+        check_repair_task_status(status_wait)
+        assert status_wait.state == "done"
```

Comment:
So the task manager could not know the task is aborted yet.  It could be a follow up.